src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Mobile/Android/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.companyname.myextensionsapp" android:versionCode="1" android:versionName="1.0">
-  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" />
-  <application android:label="MyExtensionsApp"></application>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <application android:allowBackup="true" android:supportsRtl="true"></application>
```

Comment:
I think it might be useful to have a comment here indicating where min sdk and target sdk version come from now

Reply:
This comes from the Android SDK and the version you specify in your csproj